lib/generators/rolemodel/css/base/templates/app/views/styleguide/index.html.slim

```diff
@@ -1,202 +1,236 @@
-
-      .form__group
-        label.form__label Select Dropdown
-        select.form__dropdown
-          option Option 1
-          option Option 2
-          option Option 3
-
-      .form__group
-        input.form__checkbox type="checkbox"
-        label.form__label Checkbox Input
-
-      .form__group
-        input.form__radio type="radio"
-        label.form__label Radio Input 1
-        input.form__radio type="radio"
-        label.form__label Radio Input 2
-
-  section.section--cards.margin-top-xl
-    h2.styleguide-section__title Card
-
-    .card
-      span Sample content
-
-  section.section--flashs.margin-top-xl
-    h2.styleguide-section__title flash
-    .flash.flash--notice
-      span .flash.flash--notice
-
-    .flash.flash--info
-      span .flash.flash--info
-
-    .flash.flash--alert
-      span .flash.flash--alert
-
-    .flash.flash--warning
-      span .flash.flash--warning
-
-
-  section.section--tables.margin-top-xl
-    h2.styleguide-section__title Tables
-
-    .card.card--no-padding
-      h2.margin-y-md.margin-left-md Table
-
-      table.table
-        thead
-          th Column 1
-          th Column 2
-          th
-            .text-right Action
-        tbody
-          tr
-            td
-              .flex.items-center.justify-start.margin-right-lg
-                | Data 1
-            td Data 2
-            td.text-right
-              a.btn.btn--outline.btn--small Label
-          tr
-            td
-              .flex.items-center.justify-start.margin-right-lg
-                | Data 4
-            td Data 5
-            td.text-right
-              a.btn.btn--outline.btn--small Label
-
-    .card.card--no-padding.margin-y-lg
-      h2.margin-y-md.margin-left-md Form Table
-
-      form
-        table.table
-          thead
-            th Button
-            th Measure
-            th.flex.justify-end Link
-          tbody
-            tr
-              td
-                input type="radio" id="radio-1" name="radio-button" class="margin-right-sm" Button 1
-              td item/unit
-              td.flex.justify-end Other Information
-            tr
-              td
-                input type="radio" id="radio-2" name="radio-button" class="margin-right-sm" Button 2
-              td item/unit
-              td.flex.justify-end Other Information
-            tr.form__submit
-              td
-              td
-              td.flex.justify-end.items-center
-                  button.btn.btn--outline Cancel
-                  button.btn.btn--primary.margin-left-sm type="submit" Submit
+doctype html
+html
+  head
+    title Style Guide
+    = csrf_meta_tags
+    = csp_meta_tag
+    = stylesheet_pack_tag 'stylesheets', 'data-turbolinks-track': 'reload'
+    meta name="viewport" content="width=device-width, initial-scale=1"
+  body class="#{'flex' if params[:sidebar]}"
+    - if params[:sidebar]
+      = render 'shared/sidebar'
+    - else
+      = render 'shared/top_nav'
```

Comment:
Once someone chooses a layout, we might want them to actually use that in styleguide. Is there a simple way to make that migration path easier?